google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 
-# Copyright 2021 Google LLC
+# Copyright 2021-2022 Google LLC
```

Comment:
@parthea Is this the correct formatting or should it just be `2022`. It seems like GAPIC changes to the latest year.

Reply:
Take a look at @chensun comment below

Reply:
For autogenerated files it should be `# Copyright 2022 Google LLC`. For manually updated files we should leave the original copyright year untouched as mentioned by @chensun.